.github/CODEOWNERS

```diff
@@ -210,6 +199,11 @@
 #CC# /x-pack/plugins/cloud/ @elastic/kibana-core
 #CC# /x-pack/plugins/features/ @elastic/kibana-core
 #CC# /x-pack/plugins/global_search/ @elastic/kibana-core
+#CC# /src/plugins/newsfeed @elastic/kibana-core
+#CC# /src/plugins/home/public @elastic/kibana-core
+#CC# /src/plugins/home/server/services/ @elastic/kibana-core
+#CC# /src/plugins/home/ @elastic/kibana-core
```

Comment:
Same as above, I thought Kibana App owned this now, but I could be wrong?

Reply:
@timroes what do you think?

Reply:
Nope we def gave away home and sample data to Core UI that time, not sure where it went from there, but Kibana App does not own that. My understanding from the Team organization wiki page is that Design > Kibana Design owns that currently? If that's not true, most likely we should also fix the wiki page :-)

Reply:
For both of these, the logic that originally moved them to Core UI still applies, I'd rather we left them on the Core team's responsibilities and created a hiring plan around it if there isn't bandwidth to cover them.

Reply:
It's uncomfortable to be pushing more onto either team (Core or App). However, the Core UI team is simply no more and the Kibana Design team has a very different set of objectives with even less engineering bandwith (almost none beyond the accessibility commitment). 

As an aside, the main home page is also likely to evolve into something more complex based upon the Getting Started project. As the ingest management capabilities evolve, there is quite likely to more dynamic, user/space/role oriented content regarding managing data.

It's a tough spot, and we're here to support things that arise where we can. Hopefully, the load from these two areas (home and overview page) is very quiet in the meantime.

Reply:
I must have gotten this confused with the ownership transition of the Management app going to the Kibana App team. I'm fine with Core team owning this plugin. AFAIK there aren't any expected large changes to this UI other than possibly removing the feature catalog (since we now have navigational search).

Reply:
@ryankeairns - not a problem at all.
I think you're correct - we agreed that these areas would be owned by `kibana-core` going forward.